stable/cluster-autoscaler/values.yaml

```diff
@@ -59,6 +59,8 @@ extraArgs:
   # skip-nodes-with-local-storage: false
   # skip-nodes-with-system-pods: true
 
+extraEnv: {}
```

Comment:
Is there any documentation you can add so that consumers can figure out how to use this?